proof-of-file-existence/tutorial/01-intro.md

```diff
@@ -0,0 +1,35 @@
+---
+order: 0
+---
+
+# Proof of File Existence
+
+A Proof of File Existence can help prove the authenticity of a certain file at a particular time.
+
+## How can we create a Proof of File Existence?
+
+Creating a proof of file existence can be done performing a SHA256 hash of the file or document you wish to prove exist. We can [assume](https://stackoverflow.com/questions/4014090/is-it-safe-to-ignore-the-possibility-of-sha-collisions-in-practice) that the output of a SHA256 hash function will be unique for each input, and won't be prone to collisions.
+
+The SHA256 hash of the file is also known as a checksum. When uploading the checksum to the blockchain, we create a timestamped proof that we know of a file whose hashed content equals the checksum value. This way, we are able to keep contents of file private while proving existence and ownership of the file.
+
+## PoFE use cases
+
+- Timestamping documents
+
+    You can take any document and submit the hash as a Proof of File Existence has on the blockchain. That way, you can prove that you had the file at the time when you submitted the block by sharing the file and letting others calculate the checksum. Afterwards, a link to the transaction containing that hash can be shared, proving existence of the file at the time the transaction was performed
```

Comment:
You can take any document and submit the hash as a Proof of File Existence ~~has~~ on the blockchain.